packages/dds/tree/api-report/tree.public.api.md

```diff
@@ -580,6 +580,7 @@ export class TreeViewConfiguration<const TSchema extends ImplicitFieldSchema = I
 
 // @public @sealed
 export interface TreeViewEvents {
+    // @deprecated
```

Comment:
Let's hold off on deprecating this for now. I'm not sure if we should deprecate something in the public API when the only alternative is an alpha API. Some users might not want to participate in alpha APIs, and we would be forcing them to choose between two evils. 

If we change our mind, we can easily deprecate later, no harm done, and we will at the very latest deprecate when the other event becomes public.